Podfile

```diff
@@ -73,3 +73,21 @@ end
 target 'TestAppExtension' do
   tests_main
 end
+
+target 'TestApp_Swift (tvOS)' do
+  tests_main
+end
+
+target 'TestApp_Objc (tvOS)' do
+  tests_main
+end
+
```

Comment:
Should we add tvOS to the platform to the top of this file? https://github.com/adobe/aepsdk-core-ios/pull/811/files#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbR2

Reply:
Sorry that link isn't taking me to a specific file. Which file?

Reply:
we have platform field at the top of the podfile L2

Reply:
Oh, you can only have one platform. I am not really sure how that affects things here to be honest.

Reply:
I think we need to remove this line and specify the platform and version for each target.

Reply:
@cdhoffmann I checked with our ACP extensions which have TvOS support, it looks like L2 is fine haveing one platform but we need to add TvOS platform and version for TvOS targets.

Reply:
I am not sure, but i think Cocoapod links the correct dependency based on the target platform. I also think specifying the version checks whether all the dependencies satisfy the condition.